internal/store/postgres/asset_repository.go

```diff
@@ -176,6 +180,10 @@ func (r *AssetRepository) GetByURN(ctx context.Context, urn string) (asset.Asset
 }
 
 func (r *AssetRepository) getWithPredicate(ctx context.Context, pred sq.Eq) (asset.Asset, error) {
```

Comment:
we can remove this function if not needed

Reply:
still used in UpsertPatchAsset API here: https://github.com/goto/compass/blob/main/core/asset/service.go#L213